biolink/__init__.py

```diff
@@ -0,0 +1,6 @@
+import git
+from ontobio.util.user_agent import get_user_agent
+
+NAME="biolink-api"
+VERSION=git.Repo().head.object.hexsha[:7]
```

Comment:
@kltm Since we don't do versioning yet, this was the most quickest way to get the git commit hash. 
Let me know if there is a better way to do this.

Reply:
This is start-time only?

Reply:
Yes, this runs only once, at startup.